Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -2173,9 +2173,22 @@ private void UpdateResourceAvailability(PerformanceCollector.MachinePerfInfo per
             }
 
 #if PLATFORM_OSX
-            Memory.PressureLevel pressureLevel = Memory.PressureLevel.Normal;
-            var result = Memory.GetMemoryPressureLevel(ref pressureLevel) == Dispatch.MACOS_INTEROP_SUCCESS;
-            resourceAvailable &= !(result && (pressureLevel > m_configuration.Schedule.MaximumAllowedMemoryPressureLevel));
+            if (!resourceAvailable)

```

Comment:
maybe leave a comment that the idea here is that even if resources are not available by the `exceededMaxRamUtilizationPercentage && underMinimumAvailableRam` metric, on macOS we can override that decision if the pressure level is under a configured threshold

Reply:
Will add a comment.